### ISSUE TITLE
Add ability to parse DMS coordinates

### ISSUE THREAD

**Issue by gmaclennan** [mapbox/csv2geojson#30]
I wrote this today after using excel to parse a bunch of DMS coordinates (and remembering that is not the first time I have done that):

http://gmaclennan.github.io/parse-dms/

Thought it might be useful to add this parsing to csv2geojson

**Comment by tmcw** [comment]
csv2geojson should already have this ability [via sexagesimal](https://github.com/mapbox/csv2geojson/blob/gh-pages/index.js#L104-L107)  - are you seeing any cases where it doesn't work?

**Comment by gmaclennan** [comment]
ha! I did not find sexagesimal when I searched npm, the libraries I found didn't quite do what I wanted. At least I learnt something tackling this on my own (rather pointlessly it turns out) :smiley: 

There is a side-case that sexagesimal does not get, I just got sent a collection of coordinates written as 59 12’ 7.7’’W , 02 15’39.6’’N (that's two single curly quotes for the seconds). I have also seen degrees written with º rather than ° (often by me because I forget where it is and just use option-0 on my mac).

Useful if I send a pull-request to handle these edge cases?

PS. missed a chance to call it "sexaguessimal"
